README.md

```diff
@@ -203,6 +211,16 @@ If a `telemetry` option is provided, ContainerPilot will expose a [Prometheus](h
 
 Details of how to configure the telemetry endpoint and how the telemetry endpoint works can be found in the [telemetry README](https://github.com/joyent/containerpilot/blob/master/telemetry/README.md).
 
+#### Tasks (Optional):
+
+Tasks are commands that are run periodically. They are typically used to perform housekeeping such as incremental back-ups, or pushing metrics to systems that cannot collect metrics through service discovery like Prometheus.
+
+Tasks are defined the following properties:
+
+- `command` is the executable (and its arguments) that will run when the task executes.
+- `frequency` is the time between executions of the task. Supports milliseconds, seconds, minutes. The frequency must be a positive non-zero duration with a time unit suffix. (Example: `60s`) Valid time units are `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h`.
```

Comment:
I don't think @misterbisson argues that it shouldn't be adjustable, but that if we fire the task at ns or µs resolution we're not going to have good results. If for no other reason than that it's going to take that long to launch the task executable and wire it up.